### ISSUE TITLE
Module '"./XX"' has no exported member 'type'.

### ISSUE THREAD

**Issue by hongye007** [sql-formatter-org/sql-formatter#235]
1、yarn add 
2、wrapper
``` import { format as sqlFormatter } from 'sql-formatter'

export const formatSql = (sql: string) => {
  return sqlFormatter(sql, {
    language: 'sql',
    useTabs: false,
    tabWidth: 2,
    keywordCase: 'lower',
    multilineLists: 'expressionWidth',
    expressionWidth: 100
  })
} 
```

3、use
`sql = formatSql(input)`

4、yarn build:prod or yarn serve then got error

code like : import { type XX, YY } from CC will got a error

detail:
1:10 Module '"./token"' has no exported member 'type'.
[0]   > 1 | import { type Token } from './token';
[0]       |          ^
[0]     2 | export declare type ParamItems = {
[0]     3 |     [k: string]: string;
[0]     4 | };

2:10 Module '"./token"' has no exported member 'type'.
[0]     1 | import * as regexFactory from './regexFactory';
[0]   > 2 | import { type Token, TokenType } from './token';
[0]       |          ^
[0]     3 | export declare const WHITESPACE_REGEX: RegExp;
[0]     4 | /** Struct that defines how a SQL language can be broken into tokens */
[0]     5 | interface TokenizerOptions {

**Comment by nene** [comment]
How are you importing sql-formatter? Also which exact version of sql-formatter are you using?

Looks like you're importing a typescript file but trying to interpret is as plain JavaScript.

**Comment by hongye007** [comment]
@nene It seems ok if switch sql-formatter version to 5.1.2

**Comment by nene** [comment]
This is suspicious. There is no 5.1.2 version of sql-formatter: https://www.npmjs.com/package/sql-formatter?activeTab=versions

Are you perhaps just using the code form this repository directly? Not through NPM.

**Comment by nene** [comment]
OK. This makes more sense. 6.1.2 is essentially the previous release. 6.1.3 & 6.1.4 are broken.

**Comment by nene** [comment]
Aha... I see now. Which version of TypeScript are you using?

**Comment by hongye007** [comment]
> 

Typescript Version：4.2.3

**Comment by hongye007** [comment]
> Hmm... sql-formatter is compiled with typescript 4.3.5. This shouldn't make that much of a difference.
> 
> Strangely though I can't find any mention of the `import { type Foo }` syntax from TypeScript documentation. Perhaps we have mistakenly used some unsupported syntax... will investigate.

So, What made you make those changes from version 6.1.3?

**Comment by nene** [comment]
> So, What made you make those changes from version 6.1.3?

This just happened as part of some refactoring. Had no idea it was invalid syntax. It passed all TypeScript compiler and ESLint checks just fine.

But yeah, looks like we've indeed stumbled on invalid syntax. Tried it here, and got the same error: https://stackblitz.com/edit/typescript-if3qc9?file=index.ts,foo.ts

I'll fix the syntax and do a new release.
